### DISCUSSION TITLE
Buffer settings for streaming over the net?

### DISCUSSION THREAD

**Discussion by Onegin0** [ExistentialAudio/BlackHole#376] [Q&A]
I'm trying to remote into a Mac in another town and both see and hear it, so I can remotely access mixes live.  Connection is 200 Mbps up/down fibre.

I'm using Remotix for the screen connection.  They recommended installing Blackhole on the remote Mac so it transmits audio; works great.

I am however getting random clicks / speed-changes to the audio.    

I got out Xcode, changed the buffer to 1024 and recompiled - still doing it.  Is there a recommended buffer setting for this sort of peer-to-peer connection?   

Thanks - Jon

**Comment by devinroth** [comment]
Speed changes would definitely have nothing to do with BlackHole.

Potential the clicks.

Run a test using your microphone and see if the same issues come up.

Devin

**Reply by Onegin0** [reply]
It sounds like the sort of artifact that happens on a zoom /whatsapp when the audio 'catches up' from a bad moment - the packets are delayed and all come in a rush...   

Not sure what you mean re microphone?  I'm not near the remote Mac to plug anything into it.

Thanks - Jon

**Comment by devinroth** [comment]
Well I’m 99% sure none of these bugs are related to BlackHole not working properly. Must be the other app.

Devin

**Reply by Onegin0** [reply]
So are you saying audio should stream across the internet (not a local LAN) with the default zero buffer without glitching?

**Comment by devinroth** [comment]
I’m saying that BlackHole output to input should be glitch free. I know nothing about the streaming part since that’s got nothing todo with BlackHole and sounds like that is where the errors are happening.

Devin

**Reply by Onegin0** [reply]
So under what circumstances does changing the buffer make sense?

**Comment by devinroth** [comment]
Which buffer where?

Devin

**Reply by Onegin0** [reply]
https://github.com/ExistentialAudio/BlackHole/wiki/Adjust-Driver-Latency

**Comment by devinroth** [comment]
No that won’t fix your problems.

Devin

**Reply by Onegin0** [reply]
Out of interest, what does the latency field do - simply change the latency reported to the DAW?

**Comment by devinroth** [comment]
Yup and add the appropriate delay.

Devin
